tests/TestCase/Log/LogTest.php

```diff
@@ -471,7 +471,7 @@ public function testScopedLoggingStrict()
             'path' => LOGS,
             'levels' => ['notice', 'info', 'debug'],
             'file' => 'debug',
-            'scopes' => false,
+            'scopes' => null,
```

Comment:
Why wouldn't this just be an empty array?

Reply:
Because empty array means capture all scopes :stuck_out_tongue_closed_eyes: .

Reply:
Of course.